packages/pouchdb-adapter-asyncstorage/src/bulk_docs.js

```diff
@@ -53,35 +54,59 @@ export default function (db, req, opts, callback) {
         })
       }
 
-      let binData
+      let resolveBinaryData;
```

Comment:
[Travis](https://travis-ci.org/stockulus/pouchdb-react-native/jobs/178211783#L3442) failed due to the lint errors (semicolons are forbidden in [`standard`](https://github.com/feross/standard)). 

You can check locally via

```sh
$ npm run standard
```

and full test suite via

```sh
$ npm test
```